### PR TITLE
Simplify kafka client bootstrap server names and ports config updates

### PR DESCRIPTION
**Pull request by akrambek**

### CODE REVIEW

File: src/reference/config/bindings/binding-tls.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ A list of reference names for the Vault signer certificate.
 
 ### options.trustcacerts
 
-> `boolean`
+> `boolean` Default value is `true`
```
Comment (jfallows) [inline src/reference/config/bindings/binding-tls.md:141]: ...when `trust` is `null`

**Review by jfallows — CHANGES_REQUESTED** [review]

**Review by jfallows — DISMISSED** [review]

**Review by jfallows — APPROVED** [review]

File: src/reference/config/bindings/binding-tls.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ A list of reference names for the Vault signer certificate.
 
 ### options.trustcacerts
 
-> `boolean`
+> `boolean` Default value is `true` when trust is `null`
```
Comment (vordimous) [inline src/reference/config/bindings/binding-tls.md:141]:
```suggestion
> `boolean` | Default: `true` when trust is `null`
```
A small change to align with the other `Default` definitions

File: src/reference/config/bindings/binding-tcp.md [hunk 3]
```diff
@@ -15,9 +15,6 @@ Zilla runtime tcp binding.
 tcp_server:
   type: tcp
   kind: server
-  options:
-    host: 0.0.0.0
-    port: 12345
```
Comment (vordimous) [inline src/reference/config/bindings/binding-tcp.md:20]: These don't need to be removed from the global sample unless they are being removed from the spec. For each binding, we want a description and Full example code block. We will update the examples to use the new feature but reference docs can describe everything.

File: src/reference/config/bindings/binding-kafka.md [hunk 4]
```diff
@@ -74,6 +74,7 @@ Conditional routes based on `kafka` `topic` names are used to route these networ
   - [sasl.mechanism\*](#sasl-mechanism)
   - [sasl.username](#sasl-username)
   - [sasl.password](#sasl-password)
+- [options.servers](#options-servers)
```
Comment (vordimous) [inline src/reference/config/bindings/binding-kafka.md:77]: Can you update the order of the ToC or the property section so they are in the same order. 
`### options.servers` is before `### options.sasl`, for instance.

**Review by vordimous — APPROVED** [review]

**Pull request merged**